docs/configurations.md

```diff
@@ -1,53 +1,70 @@
 # ScalarDB Configurations
 
-This document describes the configurations for ScalarDB.
+This page describes the possible configurations for ScalarDB.
```

Comment:
`available` is often used for configurations below. How about using `available` instead of `possible` here as well? I don't have a strong opinion on it, though.

Reply:
I'm OK with using `available` since it sounds stronger and more confident than `possible`. Thanks!